examples/unstructured_2d_dgsem/elixir_shallowwater_dirichlet.jl

```diff
@@ -0,0 +1,65 @@
+
+using Downloads: download
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the shallow water equations with a continuous
+# bottom topography function (set in the initial conditions)
+
+equations = ShallowWaterEquations2D(1.0, H0=3.0)
+
+initial_condition = initial_condition_well_balancedness
+
+boundary_condition_constant = BoundaryConditionDirichlet(initial_condition)
+boundary_condition = Dict( :OuterCircle => boundary_condition_constant )
+
+###############################################################################
+# Get the DG approximation space
+
+volume_flux = (flux_wintermeyer_etal, flux_nonconservative_wintermeyer_etal)
+solver = DGSEM(polydeg=4, surface_flux=(flux_hll, flux_nonconservative_fjordholm_etal),
+               volume_integral=VolumeIntegralFluxDifferencing(volume_flux))
+
+###############################################################################
+# This setup is for the curved, split form well-balancedness testing
+
+# Get the unstructured quad mesh from a file (downloads the file if not available locally)
+default_mesh_file = joinpath(@__DIR__, "mesh_outer_circle.mesh")
+isfile(default_mesh_file) || download("https://gist.githubusercontent.com/andrewwinters5000/9beddd9cd00e2a0a15865129eeb24928/raw/be71e67fa48bc4e1e97f5f6cd77c3ed34c6ba9be/mesh_outer_circle.mesh",
```

Comment:
yeah, I thought about this too. Maybe adding a folder of mesh files to the `HOHQMesh.jl` repo or something. It wouldn't increase the size of that repo much as it is only about 5 ASCII files. Although this might also be something to include @efaulhaber because I am not sure if he uses gists for the p4est mesh solver

Reply:
I wouldn't add it to the HM repo, but rather have a dedicated repo for it such that we can commit blobs without regard to the total repo size - but yes, in that case also for p4est and maybe even for the `IndicatorNeuralNetwork`?. But as I said, this was more of a general remark; at the moment my pain threshold is not yet reached, so feel free to resolve this without further action :-)

Reply:
Groovy, the topic of where to store the mesh files is as old as the `UnstructuredMesh2D` solver itself (see #500 ) so we can keep the gist strategy for now.